bin/activate

```diff
@@ -0,0 +1,76 @@
+# This file must be used with "source bin/activate" *from bash*
```

Comment:
I can't stress this enough: Don't add your virtualenv directories/files to your git repo!